fundamentals/tracks.md

```diff
@@ -0,0 +1,336 @@
+from skimage import data
+
+hubble_image = data.hubble_deep_field()
+
+tracks_data = [
+    [1, 0, 236, 0],
+    [1, 1, 236, 100],
+    [1, 2, 236, 200],
+    [1, 3, 236, 500],
+    [1, 4, 236, 1000],
+    [2, 0, 436, 0],
+    [2, 1, 436, 100],
+    [2, 2, 436, 200],
+    [2, 3, 436, 500],
+    [2, 4, 436, 1000],
+    [3, 0, 636, 0],
+    [3, 1, 636, 100],
+    [3, 2, 636, 200],
+    [3, 3, 636, 500],
+    [3, 4, 636, 1000]
+]
+
+with napari.gui_qt():
+    viewer = napari.view_image(test_im, name='image')
+    viewer.add_tracks(tracks_data, name='tracks')
+
+```
+
+![image]({{ '/assets/tutorials/tracks_simple_demo.gif' | relative_url }})
+
+## arguments of view_tracks and add_tracks
+
+Both `view_tracks` and `add_tracks` have the following doc strings:
+
+```python
+"""
+Parameters
+----------
+data : array (N, D+1)
+    Coordinates for N points in D+1 dimensions. ID,T,(Z),Y,X. The first
+    axis is the integer ID of the track. D is either 3 or 4 for planar
+    or volumetric timeseries respectively.
+properties : dict {str: array (N,)}, DataFrame
+    Properties for each point. Each property should be an array of length N,
+    where N is the number of points.
+graph : dict {int: list}
+    Graph representing associations between tracks. Dictionary defines the
+    mapping between a track ID and the parents of the track. This can be
+    one (the track has one parent, and the parent has >=1 child) in the
+    case of track splitting, or more than one (the track has multiple
+    parents, but only one child) in the case of track merging.
+    See examples/tracks_3d_with_graph.py
+color_by: str
+    Track property (from property keys) by which to color vertices.
+tail_width : float
+    Width of the track tails in pixels.
+tail_length : float
+    Length of the track tails in units of time.
+colormap : str
+    Default colormap to use to set vertex colors. Specialized colormaps,
+    relating to specified properties can be passed to the layer via
+    colormaps_dict.
+colormaps_dict : dict {str: napari.utils.Colormap}
+    Optional dictionary mapping each property to a colormap for that
+    property. This allows each property to be assigned a specific colormap,
+    rather than having a global colormap for everything.
+name : str
+    Name of the layer.
+metadata : dict
+    Layer metadata.
+scale : tuple of float
+    Scale factors for the layer.
+translate : tuple of float
+    Translation values for the layer.
+opacity : float
+    Opacity of the layer visual, between 0.0 and 1.0.
+blending : str
+    One of a list of preset blending modes that determines how RGB and
+    alpha values of the layer visual get mixed. Allowed values are
+    {'opaque', 'translucent', and 'additive'}.
+visible : bool
+    Whether the layer visual is currently being displayed.
+
+Returns
+-------
+layer : napari.layers.Tracks
+    The newly-created tracks layer.
+"""
+```
+
+## tracks data
+
+The input data to the tracks layer must be an NxD numpy array
```

Comment:
Should we change this now that the `Tracks` layer can accept a list, dataframe or array for `data`?

Reply:
Great point. Fixed!